### PR TITLE
Added Odos_V2 dex to dex.trades

### PR DESCRIPTION
**Pull request by ARDev097**
Added trade data for `odos_v2` on optimism.

### CODE REVIEW

File: models/odos/optimism/odos_v2_optimism_trades.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{{ config(
+    alias = 'trades'
+    ,schema = 'odos_v2_optimism'
```
Comment (Hosuke) [inline models/odos/optimism/odos_v2_optimism_trades.sql:3]: Hi @ARDev097,
have you added `odos_v2_optimism.trades` into `odos_optimism.trades` roll-up spell?

Comment (ARDev097) [reply models/odos/optimism/odos_v2_optimism_trades.sql:3]: Hello @Hosuke ,
I have added `odos_v2_optimism.trades` into `odos_optimism.trades`.

File: models/odos/optimism/odos_optimism_trades.sql [hunk 2]
```diff
@@ -1,103 +1,45 @@
 {{ config(
-    schema = 'odos_optimism'
-    ,alias = 'trades'
-    ,materialized = 'incremental'
-    ,file_format = 'delta'
-    ,incremental_strategy = 'merge'
-    ,unique_key = ['tx_hash', 'evt_index']
-    ,incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
-    )
+        alias = 'optimism_trades'
```
Comment (Hosuke) [inline models/odos/optimism/odos_optimism_trades.sql:2]: We still need the schema here:
```suggestion
        schema = 'odos_optimism'
        , alias = 'trades'
```

**Comment by couralex6** [comment]
Same thing here, `_sources.yml` files now live under `sources/`

**Pull request closed without merging**